modules/nlu/src/backend/api.ts

```diff
@@ -118,6 +118,64 @@ export const registerRouter = async (bp: typeof sdk, app: NLUApplication) => {
       return mapError({ botId, lang, error }, res)
     }
   })
+
+  addTrainingServiceRoutes(router, app, mapError)
+}
+
+const trainingStatusSchema = Joi.object().keys({
+  status: Joi.string()
+    .valid(['idle', 'done', 'needs-training', 'training-pending', 'training', 'canceled', 'errored'])
+    .required(),
+  progress: Joi.number()
+    .default(0)
+    .min(0)
+    .max(1)
+    .required()
+})
+
+const addTrainingServiceRoutes = (
+  router: sdk.http.RouterExtension,
+  app: NLUApplication,
+  errorMapper: (err: { botId: string; lang: string; error: Error }, res: Response) => Response
+) => {
+  const repo = app.repository
+  const SUBROUTE_NAME = 'trainrepo'
+
+  router.get(`/${SUBROUTE_NAME}/trainings`, async (req, res) => {
+    const trainings = await repo.getAll()
+    res.status(200).json(trainings)
```

Comment:
what's up with `.json(trainings)` ? Does it work ? I believe just `res.send(trainings)` will send a JSON payload \+ a status code 200.